### ISSUE TITLE
NDIlib import fails

### ISSUE THREAD

**Issue by clankill3r** [maybites/blender.script.spout#48]
From the console:

```
ndi-python is already installed with the same version as the provided wheel. Use --force-reinstall to force an installation of the wheel.
Info: Testing Package NDIlib import..
Warning: Package should be available but cannot be found, check console for detailed info. Try restarting blender, otherwise get in contact.
```

I did force reinstall the wheel. It did not help.

Blender 4.3.2

**Comment by maybites** [comment]
I have to switch to a different NDI-Library - for the time being NDI is not working.

**Comment by clankill3r** [comment]
Can it be that NDI_SDK_v5 is too new?

**Comment by maybites** [comment]
no, it should work. And I made it work for myself on both OSX and Win. But the used library is not well supported and doesn't provide wheels for all the different python libraries. I know of one that is better, but I have to find the time to implent it.

**Comment by maybites** [comment]
try the current master (https://github.com/maybites/TextureSharing/archive/refs/heads/master.zip) and get back to me. I am using now a new library that can be installed via pip like spout and syphon.

**Comment by clankill3r** [comment]
It works on OSX! I can not remember if I had this problem on windows or OSX. I can not test that earlier then somewhere end of next week tho
